resilience4j-circuitbreaker/src/test/java/io/github/resilience4j/circuitbreaker/internal/CircuitBreakerStateMachineTest.java

```diff
@@ -189,6 +190,53 @@ public void shouldTransitionToHalfOpenAfterWaitDuration() {
         assertCircuitBreakerMetricsEqualTo(-1f, 0, 0, 4, 0, 0L);
     }
 
+    @Test
+    public void shouldTransitionToHalfOpenAfterWaitInterval() {
+        CircuitBreaker intervalCircuitBreaker = new CircuitBreakerStateMachine("testName", CircuitBreakerConfig.custom()
```

Comment:
Looks like a bunch of these lines are more than 120 char, can you paginate better.

Reply:
The rest of this repo does not enforce this rule, I'll try and go through and get the parts I changed, but I won't fix the repo